SettingsList.py

```diff
@@ -2108,22 +2108,37 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'randomize_key': 'randomize_settings',
         },
     ),
-    Checkbutton(
+    Combobox(
         name           = 'shuffle_song_items',
-        gui_text       = 'Shuffle Songs with Items',
+        gui_text       = 'Shuffle Songs',
+        default        = 'song',
+        choices        = {
+            'song':    'Song Locations',
+            'dungeon': 'Dungeon Rewards',
+            'any':     'Anywhere',
```

Comment:
Just my opinion but 'song' should probably be twice as likely to be picked in randomized main rules.